about/templates/about/index.html

```diff
@@ -5,6 +5,7 @@
 
 {% block head %}
 <link rel="stylesheet" href="{% static 'about.css' %}" />
+<link rel="stylesheet" href="/main/static/base.css"/>
```

Comment:
You've included this stylesheet incorrectly. See the line right above to see how it should've been done. Also, even if the change was done correctly you are wrong to do it, since base.css is already included in base.html, from which index.html is extended. Basically, delete this line.